src/app/sessions/page.tsx

```diff
@@ -16,21 +16,32 @@ import { SelectSessionList } from "@/components/booking/SelectSessionList/Select
 import { CountIndicator } from "@/components/CountIndicator";
 import { Heading } from "@/components/Heading";
 import { Button } from "@/components/ui/button";
+import { useUser } from "@/hooks/query/getUser";
 import { MEMBER_MAX_SESSIONS, NON_MEMBER_MAX_SESSIONS } from "@/lib/constants";
 import { useCartStore } from "@/stores/useCartStore";
 
-const remainingSessions = 11;
-const isMember = true;
-const firstName = "David";
-
-const maxSessions = isMember ? MEMBER_MAX_SESSIONS : NON_MEMBER_MAX_SESSIONS;
+//const remainingSessions = 11;
+//const isMember = true;
+//const firstName = "David";
 
 export default function SelectSessionPage() {
   const { push } = useRouter();
 
   const sessionsSelected = useCartStore((state) => state.cart.length);
-
+  const { data, isLoading } = useUser("123");
```

Comment:
nice! you can fetch the current user session through the useSession() hook provided by next-auth

I believe session.user returns the user object defined under typers/next-auth